lib/routes/phoneCodeRoutes.js

```diff
@@ -26,8 +26,7 @@ function addOrUpdateCustomer (req) {
 
       return add(req.body)
     })
-    .then(customer => Promise.all([getEditedData(customer.id), getCustomerById(customer.id)]))
-    .then(([customerEditedData, customerOriginalData]) => selectLatestData(customerOriginalData, customerEditedData))
+    .then(customer => getById(customer.id))
     .then(customer => {
```

Comment:
I added this logic recently, `addOrUpdateCustomer` provides customer info to the machine, the goal  was to update the customer data with edited or manually entered data which is stored in a specific table. Removing this would ignore that data.
However, If this `getById` is part of the solution for this card, maybe it has some computation that we could reuse in `get`

Reply:
Yup, the changes I made to `getById()` include getting that manually entered customer data -- it's the `assignCustomerData()`.

Maybe we can refactor and simplify this further? I'll look into it too.

Reply:
Given our discussion, what do you think is the best thing to do?

---

@RafaelTaranto said that `getById()` is the right function to get customer info for the machine. However, we were using `getCustomerById()` previously and the machine was working fine -- except for the custom info requests; `getCustomInfoRequestsData()` should be enough for that.

Plus `getById()` adds info that is seemingly unnecessary to the machine (because `getCustomerById()` doesn't and the machine still worked fine).

Should I try to cut this bit of unnecessary server-machine traffic?